ocm-container.sh

```diff
@@ -120,6 +120,12 @@ if [ -d "${OPS_UTILS_DIR}" ]; then
 "
 fi
 
+### mount a scratch dir
+if [ -n "$SCRATCH_DIR" ]
+then
+  SCRATCH_DIR_MOUNT="-v ${SCRATCH_DIR}:/home/scratch/"
```

Comment:
Is this intended to be `/home/scratch` or `~/scratch` (/home/$USER/scratch)?

Reply:
I mounted it there locally for a reason I can't remember.
I guess `~/scratch` makes more sense. We are mounting the utils dir under `~/` so let's move it there.
I will update the branch.